backend/src/controllers/projects/closeout.js

```diff
@@ -2,72 +2,20 @@ const useController = require("@controllers/useController");
 const model = require("@models/projects/closeout");
 const what = { single: "project", plural: "projects" };
 const controller = useController(model, what);
-const { getRealmRoles } = require("@facilities/keycloak");
 
-/**
```

Comment:
why is this removed

Reply:
you'll have to be specific - which thing are you asking about in this file?

Reply:
It seemed like a great idea at the time to remove comments here, but on second thought, they could help with this feature later. I'll restore 'em

Reply:
this one is in better shape now, and should be easier to read too.